project/Game.py

```diff
@@ -137,7 +153,15 @@ def quit_question(self, team, password):
         return True
 
     def answer_question(self, team, answer):
-        return True
+        if self.team.currentLandmark.answer is not answer:
+            self.team.add_penalty(1)
+            return False
+        else:
+            pointsToAdd = self.team.currentLandmark.pointValue -\
```

Comment:
Again, currentLandmark is just an integer index.
To continue a line the \ is generally frowned upon. It's better to wrap the statement in ()